vagrant/Vagrantfile

```diff
@@ -54,3 +54,5 @@ Vagrant.configure("2") do |config|
     end
 end
 
+private_vagrantfile = File.expand_path('../Vagrantfile.private', __FILE__)
```

Comment:
Is this really looking for SRCDIR/vagrant/Vagrantfile.private and not for SRCDIR/Vagrantfile.private?